src/components/event/EventHeader.astro

```diff
@@ -12,6 +12,25 @@ if (frontmatter.end === undefined) {
 }
 const start = frontmatter.start;
 const end = frontmatter.end;
+const locations = frontmatter.locations || [];
+
+function getLinkIcon(link) {
+    if (/^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.?be)\/.+$/gi.test(link)) {
+        return 'fa-youtube';
+    } else if (/^(https?:\/\/)?(www\.)?zoom\.us\/.+$/gi.test(link)) {
+        return 'fa-zoom-us';
+    } else if (/^(https?:\/\/)?(www\.)?bilibili\.com\/.+$/gi.test(link)) {
+        return 'fa-bilibili';
+    } else if (/^(https?:\/\/)?(www\.)?github\.com\/.+$/gi.test(link)) {
+        return 'fa-github';
+    } else if (/^(https?:\/\/)?(www\.)?linkedin\.com\/.+$/gi.test(link)) {
+        return 'fa-linkedin';
+    } else if (/^(https?:\/\/)?(www\.)?twitter\.com\/.+$/gi.test(link)) {
+        return 'fa-twitter';
+    } else if (/^(https?:\/\/)?(www\.)?(docs\.google\.com)\/.+$/gi.test(link)) {
```

Comment:
slideshare would also be a nice case to handle

Reply:
Sure, added it here: https://github.com/nf-core/website/pull/2376/commits/7c7120d341a289eec67e1eb12e277b59f9ff10cf